tests/TestCase/Middleware/ThrottleMiddlewareTest.php

```diff
@@ -0,0 +1,287 @@
+<?php
+namespace Muffin\Throttle\Test\TestCase\Middleware;
+
+use Cake\Cache\Cache;
+use Cake\Core\Configure;
+use Cake\Http\Response;
+use Cake\Http\ServerRequest;
+use Cake\TestSuite\TestCase;
+use Muffin\Throttle\Middleware\ThrottleMiddleware;
+use StdClass;
+
+class ThrottleMiddlewareTest extends TestCase
+{
+    /**
+     * setUp method
+     *
+     * @return void
+     */
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->skipIf(version_compare(Configure::version(), '3.3') == -1 ? true : false);
```

Comment:
You have used `setConfig()` in the middleware which is not available in `3.3`, so need to bump this check to `3.4`.